recipe/build.sh

```diff
@@ -230,3 +241,10 @@ bazel clean
 
 # This was only needed for protobuf_python
 rm -rf $PREFIX/include/python
+
+# hmaarrfk -- 2023/12/30
+# This logic should be safe to keep in even when the underlying issue is resolved
+# https://github.com/conda-forge/cuda-nvcc-impl-feedstock/issues/9
+if [[ -x ${BUILD_PREFIX}/nvvm/bin/cicc ]]; then
+    rf ${BUILD_PREFIX}/bin/cicc
```

Comment:
Is it `rm`? I got:
```
+ rf /home/conda/feedstock_root/build_artifacts/tensorflow-split_1704582294987/_build_env/bin/cicc
/home/conda/feedstock_root/build_artifacts/tensorflow-split_1704582294987/work/conda_build.sh: line 245: rf: command not found
```

Reply:
Correct. It should be rm
```suggestion
    rm ${BUILD_PREFIX}/bin/cicc
```